### ISSUE TITLE
Can you please share an example input?

### ISSUE THREAD

**Issue by glennliu** [sayands/sgaligner#7]
Hi

Thanks for your interesting work.

I'm trying to run the code but it seems the data generation is a bit complex. I'm at the step of generating ```labels.instances.align.annotated.v2.ply``` using ShunCheng's code. And his repo also requires a few dependencies which are conflict with my Pytorch version.

So, can you provide a pair of processed scene graph inputs to let me quickly run your code? So I can skip the data generation at the moment.

Thanks
Chuhao

**Comment by sayands** [comment]
Hello, 

1. For now, I have added two example scenes data in the `.npy` format which is expected by `preprocess.py` to `example_data/` folder. I will also add the code in a while, however, the idea is just porting it in the `.npy` format after generating `labels.instances.align.annotated.v2.ply` for faster processing.

2. `scannet8_relationships.txt` contains the 8 relationships predicted by [SceneGraphFusion](https://github.com/ShunChengWu/SceneGraphFusion). I have uploaded the file to the pretrained checkpoint GDrive (link in README). However, if you want to use `3RScan` GT relationships instead, use the `relationships.txt` provided by the dataset (just change `model:rel_dim` parameter to 41 in this case).

Hope this helps, let me know in case of more issues.
